pkg/network/tracer/tracer_classification_linux_test.go

```diff
@@ -32,35 +31,6 @@ func testProtocolClassificationInner(t *testing.T, params protocolClassification
 		params.preTracerSetup(t, params.context)
 	}
 
-	var kStatsPre map[string]int64
-	if m, _ := tr.getStats(kprobesStats); m != nil {
-		if m := m["kprobes"]; m != nil {
-			kStatsPre = m.(map[string]int64)
-		}
-	}
-
-	t.Cleanup(func() {
-		if kStatsPre != nil && t.Failed() {
-			var kStatsPost map[string]int64
-			if m, _ := tr.getStats(kprobesStats); m != nil {
-				if m := m["kprobes"]; m != nil {
-					kStatsPost = m.(map[string]int64)
-				}
-				if kStatsPost == nil {
-					return
-				}
-				for k, pre := range kStatsPre {
-					if !strings.HasSuffix(k, "_misses") {
-						continue
-					}
-					if post, ok := kStatsPost[k]; ok && pre != post {
-						t.Logf("kprobe stat %s differs pre=%v post=%v", k, pre, post)
```

Comment:
It should be possible to still do this in this test, by directly calling `pkg/ebpf.GetProbeStats()`

Reply:
Yes, but this was added by me for some debugging back when we were trying to figure out failing usm tests. Its not needed now.